frontend/app/routes/protected/person-case/@types.d.ts

```diff
@@ -33,6 +33,12 @@ declare module 'express-session' {
         registrationNumber: string;
         document: File;
       };
+      personalInformation?: {
```

Comment:
Nitpick
```suggestion
      personalInformation: {
```

Reason: everything in this session slot is already marked as `Partial<>`.

Reply:
Note that the sub-properties (`firstNamePreviouslyUsed`, `lastNameAtBirth`, etc) are not included in this, so you still have to make them required or optional as needed.